modules/devkit-support/src/main/java/org/mule/security/oauth/processor/BaseOAuth2AuthorizeMessageProcessor.java

```diff
@@ -118,15 +119,16 @@ private void setState(Map<String, String> extraParameters, MuleEvent event)
         {
             for (AuthorizationParameter<?> parameter : params)
             {
+                String parameterName = parameter.getName();
                 Field field = null;
                 try
                 {
-                    field = this.getClass().getDeclaredField(parameter.getName());
+                    field = this.getClass().getDeclaredField(parameterName);
```

Comment:
This line prevents us from specifying an alias directly on DevKit. If the field I need to add on the map needs the name "Api-Key", java doesn't allow you to have an "Api-Key" field, since it is not a valid name. Get declared field will crash.